tests/integration/base_test.py

```diff
@@ -42,24 +56,30 @@ def popup_check(self, mentoring, item_feedbacks, do_submit=True):
 
         submit = mentoring.find_element_by_css_selector('.submit input.input-main')
 
+        scenario_title = mentoring.find_element_by_css_selector('h2.main')
+
+        question_title = mentoring.find_element_by_css_selector('h3.question-title')
+
         for index, expected_feedback in enumerate(item_feedbacks):
             choice_wrapper = choices_list.find_elements_by_css_selector(".choice")[index]
             if do_submit:
                 # click on actual radio button:
                 choice_wrapper.find_element_by_css_selector(".choice-selector input").click()
                 submit.click()
+                self.wait_until_exists('.choice-result.checkmark-incorrect,.choice-result.checkmark-correct')
             item_feedback_icon = choice_wrapper.find_element_by_css_selector(".choice-result")
-            choice_wrapper.click()
+            # choice_wrapper.click()
```

Comment:
@omarkhan This one was left on purpose, and I'm not sure what to do with it. 

This test checks whether assesment tooltips are shown. 

Commented out click is interpreted by the xblock as selecting one of options, and resets xblock to pre-assesment state (no checkmarks and so on) it also clears the popups. While on the next line we check whether said popups exist. It might be a flaw in test logic, or a new (minor) bug uncovered by new tests. 

If you think it's worth it we might ask some questions, but it looks like last commit to this file was a year ago. If not I'll remove the line.